doc/nrf/templates/mesh_model_README.rst

```diff
@@ -0,0 +1,210 @@
+   A model that extends another inherits the model's functionality and gives context to its states and messages.
+
+
+
+Persistent storage
+==================
+
+.. tip::
+   Not all models store information persistently.
+   If a model does not use persistent storage, write *None* in this section.
+
+The information about the Generic Power On state is stored persistently.
+
+.. tip::
+   Describe what information is stored persistently.
+   You can also specify where it is stored and what it is used for.
+
+
+API documentation
+=================
+
+| Header file: :file:`path/to/the/header/file`
+| Source file: :file:`path/to/the/source/file`
+
+.. tip::
+   Provide paths to the header and source files of the server model API.
+   Include also the doxygen group for the server model you are describing.
+
+
+----
+
+Client model
+************
+
+The XYZ Client model remotely controls the state of the XYZ Server model.
+
+.. tip::
+   Describe the XYZ Client model role and purpose in relation to the server model.
+   Clearly explain the differences in operation of the client model.
+   Because of the character of the client models, the description will be shorter than for the server model.
+
+
+Configuration*
+==============
+
+The following configuration parameters are associated with the XYZ client model:
+
+    * parameter 1
+    * parameter 2
+
+See :ref:`configure_application` for more information.
+
+.. tip::
+   List Kconfig options associated with the client model.
+   This section is optional, because not all models have Kconfig parameters that allow configuration.
+   Do not go into details here, but link to the parameter using the `:option:` reference.
+
+Extended models
+===============
+
+.. tip::
+   Not all models have extended models.
+   If a model does not have any extended model, write *None* in this section.
+
+None.
+
+.. tip::
+   List the models that are extended by the model you are describing.
+   A model that extends another inherits the model's functionality and gives context to its states and messages.
+
+Persistent storage
+==================
+
+.. tip::
+   Not all models store information persistently.
+   If a model does not use persistent storage, write *None* in this section.
+
+None.
+
+.. tip::
+   Describe what information is stored persistently.
+   You can also specify where it is stored and what it is used for.
+
+API documentation
+=================
+
+| Header file: :file:`path/to/the/header/file`
+| Source file: :file:`path/to/the/source/file`
+
+.. tip::
+   Provide paths to the header and source files of the client model API.
+   Include also the doxygen group for the client model you are describing.
+
+----
+
+Third model*
+************
+
+This section describes a third model available within the mesh model.
+Typically, a DFU facilitator.
+
+.. tip::
+   Use the section structure of either server or client models for this model.
+
+----
+
+Common types
+************
```

Comment:
Suggest making this one optional as well, in case there are no common types, like for the sensor models